app/src/main/kotlin/com/wire/android/ui/home/HomeViewModel.kt

```diff
@@ -28,6 +29,7 @@ class HomeViewModel
 @Inject constructor(
     private val navigationManager: NavigationManager,
     private val listenToEvents: ListenToEventsUseCase,
+    private val ongoingCalls: GetOngoingCallsUseCase,
     private val dataStore: UserDataStore,
     private val getAvatarAsset: GetAvatarAssetUseCase,
     private val getSelf: GetSelfUserUseCase,
```

Comment:
Seems like there is an issue here with having the threshold limit to 7 parameters, there is 7 parameters but it still accuses of it.